http adding endpoint to ingest data with (#8067)
opensearch-project/documentation-website: _data-prepper/pipelines/configuration/sources/http.md

Content will be added to this section.--->

## Ingestion

> Clients should send HTTP `POST` requests to the endpoint `/log/ingest`

Reviewer: ```suggestion
Clients should send HTTP `POST` requests to the endpoint `/log/ingest`.
```

> Currently, `http` only supports the JSON UTF-8 codec for incoming requests, such as `[{"key1": "value1"}, {"key2": "value2"}]`.

Reviewer: ```suggestion
`http` only supports the JSON UTF-8 codec for incoming requests, for example, `[{"key1": "value1"}, {"key2": "value2"}]`.
```

> **Example: Ingest data with cURL**

Reviewer: ```suggestion
#### Example: Ingest data with cURL
```

> The following example shows ingesting data with cURL:

Reviewer: ```suggestion
The following example shows how to use the cURL command to ingest data:
```
